src/words.ts

```diff
@@ -18,8 +18,9 @@ export const words = async (message: Message) => {
 
     const match = WORDS.checks
         .find((word) => WORDS.prepend
-            .find((prepend) => message.content.toLowerCase().includes(`${prepend} ${word.check}`)));
-
+            .find((prepend) => {
+                const msg = message.content.toLowerCase();
+                return msg.includes(`${prepend} ${word.check} `) || msg.includes(`${word.check}.`)}));
```

Comment:
Thanks 👍 

I wonder if we need to have `,` and `!` as well added to the end?

How we have `prepend`, should we have `append` too?

Reply:
yeah, maybe we can add more options to the `append`. Maybe the object structure would change to something like this:
```js
append: {
  prepend: [
      'hello', 'hey', 'hi', 'yo', 'hiya', 'whatsup', 'greetings',
      'bye', 'seeya', 'goodbye', 'good bye', 'thanks', 'whatsup',
      'look', 'you',
    ],
  postPend: [',', '.', '!', ';', '?']
}
```

Maybe with better names 😅

Reply:
I thought the same thing after i did a PR.

Reply:
Things always occur to me after my PRs too. But this is still an improvement, so I am happy to merge and have improvements come in a new PR. Up to you @AllanRegush